.github/workflows/code_review.yml

```diff
@@ -0,0 +1,118 @@
+name: Code Review
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+      - reopened
+  push:
+    branches:
+      - main
+
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  PROJECT_KEY: pagopa_pagopa-reporting-orgs-enrollment
+
+permissions:
+  id-token: write
+  contents: read
+  deployments: write
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  code-review:
+    name: Code Review
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Code Review
+        uses: pagopa/github-actions-template/maven-code-review@v1.4.2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          sonar_token: ${{ secrets.SONAR_TOKEN }}
+          project_key: ${{env.PROJECT_KEY}}
+          coverage_exclusions: "**/config/*,**/*Mock*,**/model/**,**/entity/*"
+          cpd_exclusions: "**/model/**,**/entity/*"
+
+#  smoke-test:
```

Comment:
Why are they commented out?

Reply:
All things related to integration and smoke tests currently cannot work correctly with the common workflow strategy (e.e. using the famous command `sh run_integration_test.sh ...`). This was already defined to be updated in a [separated task](https://pagopa.atlassian.net/jira/software/c/projects/PAGOPA/boards/336/backlog?view=detail&selectedIssue=PAGOPA-1142&issueLimit=100). For the moment, it must remain commented otherwise pipelines will fails.